celeri/solve_mcmc.py

```diff
@@ -337,6 +448,12 @@ def solve_mcmc(
 
     import nutpie
 
+    if model.config.segment_slip_rate_hard_bounds:
+        raise ValueError(
+            "Hard bounds on segment slip rates are not supported in MCMC solve. "
+            "Please use soft bounds with `segment_slip_rate_bound_sigma` instead."
```

Comment:
Please change `segment_slip_rate_bounds_type` to `soft` and set a value for `segment_slip_rate_bound_sigma`